### DISCUSSION TITLE
Input type for IP address

### DISCUSSION THREAD

**Discussion by amolrbhagwat** [quasarframework/quasar#12161] [Ideas / Proposals]
Basically the heading - it'd be great to have an input type for (at least) an IPv4 address.

**Comment by MilosPaunovic** [comment]
You can always use `type="number"` for your inputs, for mobile browsers to render numeric keyboard.

**Reply by amolrbhagwat** [reply]
Not just for numeric input, but a component that lets users enter 4 octets, with validation.

Something like this https://vuejsexamples.com/a-ip-input-component-built-for-vue-2/

**Reply by MilosPaunovic** [reply]
You can achieve something like that with an input mask.

**Reply by allard-potma** [reply]
I'm also interested in this, also with a MAC address input. The problem with the masks is that you can only mask for a-z letters, not the hex range from a-f.

**Reply by MilosPaunovic** [reply]
You can set up your own mask the way you like it:
[https://quasar.dev/vue-components/input#using-third-party-mask-processors](https://quasar.dev/vue-components/input#using-third-party-mask-processors)

**Comment by allard-potma** [comment]
@amolrbhagwat Check out this [CodeSandbox](https://codesandbox.io/s/cool-goldstine-i6vj1?file=/src/pages/Index.vue). It uses vue-imask.
